docs/conf.py

```diff
@@ -55,8 +56,8 @@
 
 # General information about the project.
 project = 'cmd2'
-copyright = '2010-2018, Catherine Devlin and Todd Leonhardt'
-author = 'Catherine Devlin and Todd Leonhardt'
+copyright = '2010-2019, Catherine Devlin and Todd Leonhardt'
```

Comment:
I'd recommend changing the copyright to also say "cmd2 contributors" with a link to the list of contributors on GitHub: https://github.com/python-cmd2/cmd2/graphs/contributors

Reply:
Done.